From: Gasol Wu <gasol.wu@gmail.com>
Date: Tue, 14 Jun 2016 17:01:31 +0800
Subject: Linking to shared library

---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index e4a8e04..bc7f295 100644
--- a/Makefile
+++ b/Makefile
@@ -112,8 +112,8 @@ core/desugarer.cpp: core/std.jsonnet.h
 	$(CXX) -c $(CXXFLAGS) $< -o $@
 
 # Commandline executable.
-jsonnet: cmd/jsonnet.cpp $(LIB_OBJ)
-	$(CXX) $(CXXFLAGS) $(LDFLAGS) $< $(LIB_SRC:.cpp=.o) -o $@
+jsonnet: cmd/jsonnet.cpp libjsonnet.so
+	$(CXX) $(CXXFLAGS) $(LDFLAGS) -ljsonnet -L. $< -o $@
 
 # C binding.
 libjsonnet.so: $(LIB_OBJ)
